src/AbstractActionReader.ts

```diff
@@ -73,10 +73,11 @@ export abstract class AbstractActionReader {
       } else {
         // Since the new block did not match our history, we can assume our history is wrong
         // and need to roll back
-        console.info("!! Fork detected !!")
-        console.info(`  expected: ${expectedHash}`)
-        console.info(`  received: ${actualHash}`)
-        await this.rollback()
+        console.info("!! FORK DETECTED !!")
```

Comment:
`console.info("!! FORK DETECTED, WHAT THE FORK !!")`

Reply:
Serious comment noted